docs/source/conf.py

```diff
@@ -16,9 +16,11 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+from datetime import datetime
+sys.path.insert(0, os.path.abspath('../../src'))
```

Comment:
This is unnecessary. The package is installed on ReadTheDocs and in the local tox environment so you should never need to insert the path to get this import to work. It's cruft and it will likely cause confusion at some point in the future. Please remove it.

Reply:
I've removed the path manipulation